languages/python/oso/oso/oso.py

```diff
@@ -77,3 +79,10 @@ def get_allowed_actions(self, actor, resource, allow_wildcard=False) -> list:
             actions.add(action)
 
         return list(actions)
+
+    def _print_polar_log_message(self):
+        if os.environ.get("POLAR_LOG", None):
+            print(
+                "Polar tracing enabled. Get help with "
```

Comment:
Doesn't have to get figured out as part of this PR, but should this maybe be `OSO_LOG=1` and `Oso tracing enabled.` in light of our attempts to standardize on Oso > Polar in public?

Reply:
yes!